security/PracticalSecurity.md

```diff
@@ -0,0 +1,27 @@
+### Password Manager
+It may be completely irrational, but I don't trust password managers.  I am essentially giving some company access to all my passwords under the guise of protection.  While it would be counterintuitive for a company to compromise their business by doing this, there is literally nothing stopping them.  Not to mention their code could have a security flaw in and of itself, allowing some black hat to steal all of my passwords.  I also don't save any passwords in my browser / applications (except for passwords that I don't care about like zoom or repl.it).  I don't even use my real name for things most of the time.
```

Comment:
I like this aproach as i am following similar ideology about password manager's. Very nicely written.